.gitignore

```diff
@@ -33,4 +33,5 @@ utils/regression_test.js
 
 node_modules/
 
-source/_build/
\ No newline at end of file
+source/_build/
+*.iml
```

Comment:
Where is this for?

Reply:
to ignore IntelliJ module configuration files (i included the documentation in my IntelliJ project)